languages/python/django-oso/django_oso/oso.py

```diff
@@ -36,11 +40,22 @@ def register_class(model, name=None):
         for model in app.get_models():
             register_class(model, polar_model_name(model))
 
-    # Custom registration for auth (AnonymousUser)
+    # Custom registration for auth (AnonymousUser, User)
     if apps.is_installed("django.contrib.auth"):
         from django.contrib.auth.models import AnonymousUser
 
-    register_class(AnonymousUser, name=f"django::contrib::auth::AnonymousUser")
+        # Register under `auth` app_label to match default User model, but also fully-qualified name
+        register_class(AnonymousUser, "auth::AnonymousUser")
+        register_class(AnonymousUser, "django::contrib::auth::AnonymousUser")
+
+        # Register the user model, possibly customized
+        try:
+            from django.contrib.auth import get_user_model
```

Comment:
This shouldn't be necessary, because the registration on L41 (https://github.com/osohq/oso/pull/562/files#diff-f0f374f317fb12ecab1cae1843229bf549c8f2d6c1efdf882ed4a72f0f031996R41) should also register the built in User model if it's part of an installed app.

Were you having trouble using the built-in User model in the policy?